staging/src/k8s.io/apiserver/pkg/util/feature/feature_gate.go

```diff
@@ -70,10 +70,16 @@ const (
 // FeatureGate parses and stores flag gates for known features from
 // a string like feature1=true,feature2=false,...
 type FeatureGate interface {
+	// AddFlag adds a flag for setting global feature gates to the specified FlagSet.
 	AddFlag(fs *pflag.FlagSet)
+	// Set Parses a string of the form // "key1=value1,key2=value2,..." into a
```

Comment:
I don't think you should say anything about `map[string]bool` as that's an implementation detail. I think you can echo the comment above for `FeatureGate` itself: `Set parses and stores flag gates for known features from a string like feature1=true,feature2=false,...`